packages/google_maps_flutter/google_maps_flutter_web/lib/src/convert.dart

```diff
@@ -64,7 +64,7 @@ gmaps.MapOptions _configurationAndStyleToGmapsOptions(
       configuration.zoomGesturesEnabled == false) {
     options.gestureHandling = 'none';
   } else {
-    options.gestureHandling = 'auto';
+    options.gestureHandling = 'greedy';
```

Comment:
I should have added a comment for this. With the 'auto' setting, using the scroll wheel to zoom requires holding a modifier key and the user gets an instruction like the following:
<img width="800" alt="Screenshot 2023-05-23 at 11 13 00 AM" src="https://github.com/flutter/packages/assets/602697/7e700df6-3879-49cc-a881-1ca3dfc73481">

We changed the setting to 'greedy' so that the map could be zoomed with the scroll wheel without using the modifier key.

Reply:
@elitree please, revert this for now! There's more proper web gesture handling incoming here:

https://github.com/flutter/packages/pull/3258/

Reply:
I rebased this morning, and reverted this change.